src/bindings/python/tests/test_runtime/test_async_infer_request.py

```diff
@@ -325,3 +316,122 @@ def test_shared_memory_deprecation(device, shared_flag):
     with pytest.warns(FutureWarning, match="`shared_memory` is deprecated and will be removed in 2024.0"):
         queue.start_async(input_data, shared_memory=shared_flag)
     queue.wait_all()
+
+
+@pytest.mark.skip(reason="Sporadically failed. Need further investigation. Ticket - 95967")
```

Comment:
what kind of errors it reports?

Reply:
It sporadically failed in CI some time ago.
Unfortunately, I couldn't find what exactly the error was. 
BTW, we have a ticket for this skip -- 95967.
I may try to remove skip after CF.